src/api/app/jobs/send_event_emails.rb

```diff
@@ -16,16 +17,29 @@ def perform
       subscribers = event.subscribers
       next if subscribers.empty?
       EventMailer.event(subscribers, event).deliver_now
-
-      event.subscriptions.each do |subscription|
-        Notifications::RssFeedItem.create(
-          subscriber: subscription.subscriber,
-          event_type: event.eventtype,
-          event_payload: event.payload,
-          subscription_receiver_role: subscription.receiver_role
-        )
-      end
+      create_rss_notification(event, renderer)
     end
     true
   end
+
+  private
+
+  def create_rss_notification(event, renderer)
+    description = renderer.render(
+      template: "event_mailer/#{event.template_name}",
+      layout: false,
+      format: :txt,
+      assigns: { e: event.expanded_payload, host: ::Configuration.obs_url, configuration: ::Configuration.first })
+
+    event.subscriptions.each do |subscription|
+      Notifications::RssFeedItem.create(
+        subscriber: subscription.subscriber,
+        event_type: event.eventtype,
+        event_payload: event.payload,
+        title: event.subject,
+        description: description,
```

Comment:
Do we want to save the rendering of the event? `event.payload` is all the we need to render the view for the event so we could just save `event.payload` and render on the fly?

Also this would be better named `body` or `content` rather than `description`.